parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1848,40 +1826,56 @@ loop = do
               )
           pure . join $ toList xs
 
-        configKey k p =
```

Comment:
Moved to top level

Reply:
Meta: It's a bummer that these comments on "Outdated" commits don't really show up anywhere in context now.

Reply:
That's lame. GitHub didn't follow the file renamings as well as git did.